### PR TITLE
NO-ISSUE: align cluster pool version for subsystem-tests

### PR DESCRIPTION
**Pull request by eifrach**
align cluster pool version for OCM release

- 2.7  4.14
- 2.8 4.15
- 2.9 4.15
- 2.10 4.15

all other should run on latest 4.15

### CODE REVIEW

**Comment by eifrach** [comment]
/pj-rehearse  pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/pj-rehearse  pull-ci-openshift-assisted-service-master-edge-subsystem-aws
/pj-rehearse  pull-ci-openshift-assisted-service-master-edge-subsystem-kubeapi-aws

**Comment by eifrach** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-subsystem-aws

**Comment by eifrach** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-kubeapi-aws-periodic

File: ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.7.yaml [hunk 1]
```diff
@@ -234,7 +234,7 @@ tests:
     owner: openshift-ci
     product: ocp
     timeout: 1h0m0s
-    version: "4.10"
```
[the remark this answers is not in the document]
Comment (eifrach) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.7.yaml:237]: cause  the test is running against the claimed cluster 

so when claim 4.15 then you run the subsystem against it - so the code should be good  only up to this version after that it's not supported and we dont need to test it.

if you claim unsupported version you may have test that will fail

[the remark this answers is not in the document]
Comment (danmanor) [reply ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.7.yaml:237]: What I meant is that I think it is not relevant for assisted-service, only for OCM. never mind though, this is urgent lets discuss this in other time

**Comment by eifrach** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/test clusterimageset-validate

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic

**Comment by eifrach** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic

**Comment by danmanor** [comment]
/pj-rehearse periodic-ci-openshift-assisted-service-master-edge-subsystem-aws-periodic

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.7-subsystem-aws

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-release-ocm-2.8-subsystem-aws

**Pull request closed without merging**